applications/asset_tracker_v2/tests/ui_module/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(ui_module_test)
+
+# Generate runner for the test
+test_runner_generate(src/ui_module_test.c)
+
+# Create mocks
+cmock_handle(../../src/modules/modules_common.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/event_manager.h )
```

Comment:
Use the CMake Variable `${NRF_DIR}` to refer to the nrf director. There are also other variables like `${ZEPHYR_NRFXLIB_MODULE_DIR}` for referring to the paths of the module repos.